end-end-tests/api-standards/resources/thing/2021-11-10/001-singleton.yaml

```diff
@@ -21,7 +21,7 @@ paths:
       parameters:
         - { $ref: "#/components/x-rest-common/parameters/Version" }
       responses:
-        "201":
+        "200":
```

Comment:
Non-blocking (especially as it's not part of this PR) but the description and operationId above and below should be updated to match that this is a get request